src/Controls/samples/Controls.Sample/Pages/Controls/RadioButtonGalleries/RadioButtonView.xaml

```diff
@@ -0,0 +1,40 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
```

Comment:
The test xaml copied from the [repo](https://github.com/davidbritch/dotnet-maui-samples/blob/main/UserInterface/Views/RadioButtonDemos/RadioButtonDemos/Views/RadioButtonsExplainedPage.xaml) reported in the bug